cdist/conf/type/__grub_config/manifest

```diff
@@ -0,0 +1,186 @@
+[ -f "$__object/parameter/GRUB_DEFAULT_BUTTON" ] && GRUB_DEFAULT_BUTTON=$(cat "$__object/parameter/GRUB_DEFAULT_BUTTON")
+[ -f "$__object/parameter/GRUB_TIMEOUT_BUTTON" ] && GRUB_TIMEOUT_BUTTON=$(cat "$__object/parameter/GRUB_TIMEOUT_BUTTON")
+[ -f "$__object/parameter/GRUB_TIMEOUT_STYLE_BUTTON" ] && GRUB_TIMEOUT_STYLE_BUTTON=$(cat "$__object/parameter/GRUB_TIMEOUT_STYLE_BUTTON")
+[ -f "$__object/parameter/GRUB_BUTTON_CMOS_ADDRESS" ] && GRUB_BUTTON_CMOS_ADDRESS=$(cat "$__object/parameter/GRUB_BUTTON_CMOS_ADDRESS")
+[ -f "$__object/parameter/GRUB_TERMINAL_INPUT" ] && GRUB_TERMINAL_INPUT=$(cat "$__object/parameter/GRUB_TERMINAL_INPUT")
+[ -f "$__object/parameter/GRUB_TERMINAL_OUTPUT" ] && GRUB_TERMINAL_OUTPUT=$(cat "$__object/parameter/GRUB_TERMINAL_OUTPUT")
+[ -f "$__object/parameter/GRUB_SERIAL_COMMAND" ] && GRUB_SERIAL_COMMAND=$(cat "$__object/parameter/GRUB_SERIAL_COMMAND")
+[ -f "$__object/parameter/GRUB_DISABLE_LINUX_UUID" ] && GRUB_DISABLE_LINUX_UUID=$(cat "$__object/parameter/GRUB_DISABLE_LINUX_UUID")
+[ -f "$__object/parameter/GRUB_DISABLE_RECOVERY" ] && GRUB_DISABLE_RECOVERY=$(cat "$__object/parameter/GRUB_DISABLE_RECOVERY")
+[ -f "$__object/parameter/GRUB_VIDEO_BACKEND" ] && GRUB_VIDEO_BACKEND=$(cat "$__object/parameter/GRUB_VIDEO_BACKEND")
+[ -f "$__object/parameter/GRUB_BACKGROUND" ] && GRUB_BACKGROUND=$(cat "$__object/parameter/GRUB_BACKGROUND")
+[ -f "$__object/parameter/GRUB_THEME" ] && GRUB_THEME=$(cat "$__object/parameter/GRUB_THEME")
+[ -f "$__object/parameter/GRUB_GFXPAYLOAD_LINUX" ] && GRUB_GFXPAYLOAD_LINUX=$(cat "$__object/parameter/GRUB_GFXPAYLOAD_LINUX")
+[ -f "$__object/parameter/GRUB_DISABLE_OS_PROBER" ] && GRUB_DISABLE_OS_PROBER=$(cat "$__object/parameter/GRUB_DISABLE_OS_PROBER")
+[ -f "$__object/parameter/GRUB_OS_PROBER_SKIP_LIST" ] && GRUB_OS_PROBER_SKIP_LIST=$(cat "$__object/parameter/GRUB_OS_PROBER_SKIP_LIST")
+[ -f "$__object/parameter/GRUB_DISABLE_SUBMENU" ] && GRUB_DISABLE_SUBMENU=$(cat "$__object/parameter/GRUB_DISABLE_SUBMENU")
+[ -f "$__object/parameter/GRUB_ENABLE_CRYPTODISK" ] && GRUB_ENABLE_CRYPTODISK=$(cat "$__object/parameter/GRUB_ENABLE_CRYPTODISK")
+[ -f "$__object/parameter/GRUB_PRELOAD_MODULES" ] && GRUB_PRELOAD_MODULES=$(cat "$__object/parameter/GRUB_PRELOAD_MODULES")
+[ -f "$__object/parameter/GRUB_RECORDFAIL_TIMEOUT" ] && GRUB_RECORDFAIL_TIMEOUT=$(cat "$__object/parameter/GRUB_RECORDFAIL_TIMEOUT")
+[ -f "$__object/parameter/GRUB_RECOVERY_TITLE" ] && GRUB_RECOVERY_TITLE=$(cat "$__object/parameter/GRUB_RECOVERY_TITLE")
+[ -f "$__object/parameter/GRUB_HIDDEN_TIMEOUT" ] && GRUB_HIDDEN_TIMEOUT=$(cat "$__object/parameter/GRUB_HIDDEN_TIMEOUT")
+[ -f "$__object/parameter/GRUB_HIDDEN_TIMEOUT_QUIET" ] && GRUB_HIDDEN_TIMEOUT_QUIET=$(cat "$__object/parameter/GRUB_HIDDEN_TIMEOUT_QUIET")
+[ -f "$__object/parameter/GRUB_HIDDEN_TIMEOUT_BUTTON" ] && GRUB_HIDDEN_TIMEOUT_BUTTON=$(cat "$__object/parameter/GRUB_HIDDEN_TIMEOUT_BUTTON")
+
```

Comment:
You could also simplify code by removing above lines and put that logic inside loop body below.